nntrainer/optimizers/lr_scheduler.h

```diff
@@ -92,6 +93,11 @@ class LearningRateScheduler : public ml::train::LearningRateScheduler {
    * - min_learning_rate : float
    * - decay_steps : float
    *
+   * Linear Learning rate scheduler
+   * - max_learning_rate : float
+   * - min_learning_rate : float
+   * - decay_steps : float
```

Comment:
decay_steps is stated as a float. shouldn't it be an integer?

Reply:
changed it 'positive integer'